src/main/java/com/revature/services/PersonLogic.java

```diff
@@ -0,0 +1,17 @@
+package com.revature.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.beans.factory.annotation.Autowired'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)